packages/react-native-reanimated/android/src/reactNativeVersionPatch/NativeProxyFabric/latest/com/swmansion/reanimated/NativeProxy.java

```diff
@@ -87,11 +87,19 @@ private native HybridData initHybridBridgeless(
 
   public native void performOperations();
 
+  public native void invalidateCpp();
+
   @Override
   protected HybridData getHybridData() {
     return mHybridData;
   }
 
+  @Override
+  public void invalidate() {
+    super.invalidate();
+    invalidateCpp();
```

Comment:
Shouldn't we reverse the order here?
```suggestion
    invalidateCpp();
    super.invalidate();
```